### DISCUSSION TITLE
The new Obsidian 1.4 feature "Properties" breaks my "Book search" template

### DISCUSSION THREAD

**Discussion by fgsanz** [anpigon/obsidian-book-search-plugin#77] [Q&A]
The new Obsidian 1.4 feature "Properties" breaks my "Book search" template. Does this happen to you also?

**Comment by Cryptodarktrader** [comment]
If you use a template as a base, for example from your templates folder, it does not work correctly.

but if you have the information (the data of your template) within the two fields that appear in settings in Fronmatter settings and in content settings it works correctly
![Captura](https://github.com/anpigon/obsidian-book-search-plugin/assets/145910014/3b774830-17aa-4d59-a337-973c1e24eaaa)

**Comment by writtenfool** [comment]
It works for me, but I did have to tweak my template to match what is required in Obsidian to render the variables provided by this plugin.

**Reply by AyersAuthentic** [reply]
What tweak did you need to do to fix this?

**Comment by fgsanz** [comment]
This is how I tweaked it and now it works for me:

```
---
tags:
  - book/paper
  - book/PDF
  - book/Kindle
  - book/Audible
  - book/web
title: "{{title}}"
subtitle: "{{subtitle}}"
rating: 
series: 
authors:
  - "[[{{author}}]]"
narrator: 
category:
  - Scientific journalism
  - Technical
  - Nonfiction
  - Fiction
  - Poetry
  - Biography
  - Memoir
publisher: "{{publisher}}"
published: "{{publishDate}}"
pages: "{{totalPage}}"
isbn10: "{{isbn10}}"
isbn13: "{{isbn13}}"
cover: "{{coverUrl}}"
date: "{{DATE:YYYY-MM-DD}}"
read: 
extra_file: false
parent: 
related:
---

```

**Reply by fgsanz** [reply]
In the template I have several categories and tags, and then depending on the book I easily delete (click on "X" on each) the ones that do not fit.
